### PR TITLE
Added file priority logic for JS repos

### PR DESCRIPTION
**Pull request by Saumya25mis**
This PR contains changes done to assign file priority score for the Java Script repos, 
Added a function getFileScoreJS for the Implementation of the same.

### CODE REVIEW

**Comment by pandurangpatil** [comment]
@Saumya25mis can you add the details of what changes have been done in this PR? I don't see any description

File: src/main/scala/ai/privado/audit/DataElementDiscovery.scala [hunk 1]
```diff
@@ -530,7 +574,7 @@ object DataElementDiscoveryJS {
           workbookResult += List(
             key.name,
             key.file.head.name,
-            getFileScore(key.file.name.headOption.getOrElse(Constants.EMPTY), xtocpg),
+            getFileScoreJS(key.file.name.headOption.getOrElse(Constants.EMPTY), xtocpg),
```
[the remark this answers is not in the document]
Comment (hiteshbedre) [reply src/main/scala/ai/privado/audit/DataElementDiscovery.scala:577]: Got it.

**Review by hiteshbedre — DISMISSED** [review]
LGTM

File: src/main/scala/ai/privado/audit/DataElementDiscovery.scala [hunk 2]
```diff
@@ -452,6 +453,49 @@ object DataElementDiscoveryJS {
...
+
+        val lowPriorityFolders = "(?i)routes?|test|public|assets?|static|spec|configs?".r
+        lowPriorityFolders.findFirstMatchIn(absoluteFileName) match {
+          case Some(_) => score = 0.0
+          case None    =>
+        }
+        score.toString
+      }
+      case Failure(exception) => {
+        println("Failed to calculate file score")
```
Comment (hiteshbedre) [inline src/main/scala/ai/privado/audit/DataElementDiscovery.scala:491]: We can add filename in log, good to have.

**Review by HiteshMah-Jan — DISMISSED** [review]
Looks good

File: src/main/scala/ai/privado/audit/DataElementDiscovery.scala [hunk 3]
```diff
@@ -452,6 +453,50 @@ object DataElementDiscoveryJS {
...
+        val lowPriorityFolders = "(?i)routes?|test|public|assets?|static|spec|configs?".r
+        lowPriorityFolders.findFirstMatchIn(absoluteFileName) match {
+          case Some(_) => score = 0.0
+          case None    =>
+        }
+        score.toString
+      }
+      case Failure(exception) => {
+        val fileName = absoluteFileName.substring(absoluteFileName.lastIndexOf("/") + 1)
+        println("Failed to calculate file score for " + fileName)
```
Comment (HiteshMah-Jan) [inline src/main/scala/ai/privado/audit/DataElementDiscovery.scala:492]: plz change println to log.debug - else it will show up in cli output

**Review by HiteshMah-Jan — DISMISSED** [review]

**Review by HiteshMah-Jan — APPROVED** [review]

**Pull request merged**